package.json

```diff
@@ -5,7 +5,7 @@
   "main": "bundle.js",
   "scripts": {
     "build": "webpack",
-    "start": "yarn build && jekyll serve",
+    "start": "yarn watch && jekyll serve --livereload",
```

Comment:
i think this will start jekyll serve twice, one with and one without live reload.